runtime/src/main/cpp/Exceptions.cpp

```diff
@@ -229,8 +230,23 @@ void OnUnhandledException(KRef throwable) {
   }
 }
 
+#if KONAN_REPORT_BACKTRACE_TO_IOS_CRASH_LOG
+static bool terminating = false;
```

Comment:
Why not just keep them in anonymous namespace instead of static?

Reply:
Anonymous namespaces are too C++ :)
Using `static` is mostly a matter of taste. I like `static` because it makes visibility of the function obvious and doesn't require a developer to look around.